public/app/core/components/Branding/Branding.tsx

```diff
@@ -58,8 +58,8 @@ export class Branding {
   static LoginBackground = LoginBackground;
   static MenuLogo = MenuLogo;
   static LoginBoxBackground = LoginBoxBackground;
-  static AppTitle = 'Grafana';
-  static LoginTitle = 'Welcome to Grafana';
+  static AppTitle = 'Dashboards';
+  static LoginTitle = 'Welcome to SystemLink Dashboards';
```

Comment:
Did we get Ritesh or Josh's thoughts on what text to put here? Could you maybe add a screenshot showing what this looks like in context?

Reply:
No, and the user should never encounter the login page with the auth-proxy changes that Carson is making. Do you think I should just revert this?

Reply:
Yeah, if it's not necessary let's revert it.